notice_and_comment/templates/regulations/nc-homepage.html

```diff
@@ -22,25 +23,38 @@
           <h1>Read, understand, and comment on new proposed regulations.</h1>
           <h2>EPA is soliciting comment on the following:</h2>
 
+          {% for meta in notices %}
           <div class="proposed-rule">
 
             <div class="comments-date group">
-              <div class="due">Comments are due in <strong>X days!</strong></div>
-              <div class="date">Ends September XX, 2016 at 11:59pm</div>
+
+                {% if meta.comment_state.value == comment_state.PREPUB %}
+                  <div class="date">Comment period is not open. The comment period will open once the rule is officially published in the Federal Register.</div>
+                {% elif meta.comment_state.value == comment_state.OPEN %}
+                  <div class="due">Comments are due in <strong>{{meta.days_remaining}} days!</strong></div>
+                  <div class="date">Ends {{meta.comments_close|date:"F d, Y"}}.</div>
+                {% elif meta.comment_state.value == comment_state.CLOSED %}
+                  <div class="date">As of {{meta.comments_close|date:"F d, Y"}}, we are no longer accepting comments on this rule.</div>
+                {% elif meta.comment_state.value == comment_state.NO_COMMENT %}
+                  <div class="date">This notice does not accept comments.</div>
+                {% endif %}
+
             </div>
 
             <div class="agency">
-              EPA
+                EPA
             </div>
 
             <div class="proposed-rule-title">
-              Hazardous Waste Management System; User Fees for the Electronic Hazardous Waste Manifest System and Amendments to the Manifest Regulations
+              {{meta.title}}
+            </div>
+
+            <div class="read-proposed-rule">
+                <a href="/preamble/{{ meta.document_number|dash_to_underscore }}">Read the proposed rule and write a comment <span class="fa fa-chevron-right" aria-hidden="true"></span></a>
```

Comment:
Django's `{% url %}` would make this a bit more portable -- it accounts for different mount points